src/mon/Monitor.cc

```diff
@@ -464,6 +464,7 @@ const char** Monitor::get_tracked_conf_keys() const
     "clog_to_graylog",
     "clog_to_graylog_host",
     "clog_to_graylog_port",
+    "host",
```

Comment:
Looks good to me. Seems like I forgot the "host" key. Sorry and thanks for spotting and fixing.